classes/Product.php

```diff
@@ -4468,6 +4468,27 @@ public static function duplicateTaxes($id_product_old, $id_product_new)
         return true;
     }
 
+    public static function duplicatePrices($id_product_old, $id_product_new)
```

Comment:
Hello, thanx a lot for your contribution !

Could you please add a proper phpdoc including a short description of the method ?